### ISSUE TITLE
ENIGMA-DTI needs cleaning up + documentation contains errors

### ISSUE THREAD

**Issue by josephdviviano** [TIGRLab/admin#169]
+ Documentation for ENIGMA-DTI references scripts that do not exist, etc. Investigate and synchronize the docs with the code.
+ The code itself should be cleaned up.
+ I've merged DTI_preproc with ENIGMA-DTI.

**Comment by djrotenberg** [comment]
As an extension to this we found that the existing binaries (singleSubjROI_exe and averageSubjectTracts_exe) did not provide the correct number of averaged ROIs required, which would also cause errors in the QC scripts (http://enigma.ini.usc.edu/wp-content/uploads/DTI_Protocols/ENIGMA_DTI_plots_ALL.R)

To resolve this I downloaded the latest version of the code from:
http://enigma.ini.usc.edu/wp-content/uploads/2012/06/ROIextraction_info.zip

Because this was acquired in 2015, I created a new directory under /opt/quarantine/ENIGMA-DTI/2015.01

For brevity and to test the scripts quickly I copied the two binaries identified above from the 2015.01 extraction to 2014.01 with an "_2" in their name and the master script "ENIGMA_MASTER.sh" was rewritten to reference these. 

Ideally the master script should be migrated to the 2015.01 build, referencing the original binary names and a new module file created. 

NOTE: Useful Links for Documentation
http://enigma.ini.usc.edu/wp-content/uploads/DTI_Protocols/ENIGMA_DTI_SummaryStats_QC_protocol_USC.pdf
http://enigma.ini.usc.edu/wp-content/uploads/2014/01/ENIGMA_ROI_protocol_USC.pdf

**Comment by pipitone** [comment]
Okay, I've updated the 2015.01 with the master script. It's completely untested though (hi Tina!). 

I've also reverted the 2014.01 code to not point to the _2 files. Generally, doing so is a bad idea, since a quarantine is meant to be quarantined from changes! 

Frankly, the ENGIMA code is a nightmare, and I don't trust it really, even after these changes. 

I'm leaving this issue open until Tina confirms the updated code works, but I unless someone with a lot of patience comes along, I don't think we're going to get around to cleaning up ENIGMA's messy code.
